From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Daniel Muey <dan@cpanel.net>
Date: Wed, 19 Aug 2020 11:49:47 -0500
Subject: [PATCH] Fix auto/feature test C code to not fail due to its use of
 printf

---
 auto/feature | 1 +
 1 file changed, 1 insertion(+)

diff --git a/auto/feature b/auto/feature
index 3561f59..972b12c 100644
--- a/auto/feature
+++ b/auto/feature
@@ -27,6 +27,7 @@ fi
 
 cat << END > $NGX_AUTOTEST.c
 
+#include <stdio.h>
 #include <sys/types.h>
 $NGX_INCLUDE_UNISTD_H
 $ngx_feature_incs
